addons/sms/models/res_partner.py

```diff
@@ -1,12 +1,13 @@
 # -*- coding: utf-8 -*-
 # Part of Odoo. See LICENSE file for full copyright and licensing details.
 
-from odoo import models
+from odoo import models, fields
 
 
 class ResPartner(models.Model):
     _name = 'res.partner'
     _inherit = ['res.partner', 'mail.thread.phone']
+    phone_mobile_search = fields.Char("Phone/Mobile", store=False, search='_search_phone_mobile_search')
```

Comment:
Still, you don't have to define it as it comes from the mail.thread.phone mixin inherited hereabove ...

Reply:
Oh yeah... corrected.